src/Transport/obsoletes-v3.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus
+{
+    using System;
+    using Azure.Core;
+    using Azure.Messaging.ServiceBus;
+    using Extensibility;
+
+    /// <summary>
+    /// Adds access to the Azure Service Bus transport config to the global Transport object.
+    /// </summary>
+    public static partial class AzureServiceBusTransportSettingsExtensions
```

Comment:
Looking at this again, it might be better to implement these in the originalclass file and pre-obsolete them. That way they will match [all of the other implementations](https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/blob/fa0544789f4cb9412234fc81611f1d77c45c46dd/src/Transport/AzureServiceBusTransportSettingsExtensions.cs)